src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import Header from './components/Header';
+import IllustratedMessage from './components/IllustratedMessage';
+import Button from 'components/button/Button';
+import LogoSrc from './components/assets/logo.svg';
+
+class Dashboard extends Component {
+  getBanner = () => (
+    <IllustratedMessage
+      imageSrc={LogoSrc}
+      message="You have 3 new mentorship messages!"
```

Comment:
i18n?

Reply:
Well this should not even be here. This is an example hardcoded. I guess this "banner" or "Dashboard notifications" or "dashboard messages" will be get through redux. 

I just wanted to point out how to use it.. maybe its better to do this on the story and then remove from the view for now? @crysfel